test/test_heuristics.rb

```diff
@@ -66,6 +66,16 @@ def test_pm_perl_by_heuristics
     })
   end
 
+  # Candidate languages = ["Perl", "Perl6"]
+  def test_t_perl_by_heuristics
+    assert_heuristics({
+      "Perl" => all_fixtures("Perl", "*.t"),
+      "Perl6" => ["Perl6/01-dash-uppercase-i.t", "Perl6/01-parse.t", "Perl6/advent2009-day16.t",
```

Comment:
Note that `samples/Perl6/man-or-boy.t` _doesn't_ get picked up by this heuristic but _is_ correctly classified (outside of the heuristics) as it has a vim modeline.